src/core/glossary.md

```diff
@@ -0,0 +1,29 @@
+# Glossary
+
+## Simulation Sequence Components
+
+**cache:** Dictionary of each stage and its vector of caches. Each cache tracks the last time status of a device changed in a simulation.
+
+**feed_forward:** The vertical inter-stage relationship between the binary variable status of one stage and the affected variables of the next stage.
```

Comment:
Not a relationship, it is the variable that is used as a parameter for a later stage.

Reply:
Jose recently changed all uses to `feedforward`